tests/XrdHttpTests/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+if(APPLE OR NOT BUILD_HTTP)
```

Comment:
@ccaffy You can now revert this back to just `if(NOT BUILD_HTTP)`, as I have put a workaround for issue #1772 in master that will allow the tests to run even without a proper hostname.

Reply:
Sure :)